src/Microsoft.AspNet.OData.Versioning.ApiExplorer/ReleaseNotes.txt

```diff
@@ -1 +1,2 @@
-﻿Fixed query options for non-action POST
\ No newline at end of file
+﻿Fixed query options for non-action POST
```

Comment:
Apologies, I probably wasn't clear on this. You need only replace the release notes. Since the list can get very long, the package release notes are whatever was patched in the published package. The next release on GitHub will have the rollup of everything. I moved to using a text file so it would be easier to diff the file for changes over time. You can simply SHIFT+DEL the first line. Thanks.